docs/examples.md

```diff
@@ -56,6 +56,9 @@ This page links to examples of how to implement some things with the Epic Stack.
 - [Epic Stack + Fathom Analytics](https://github.com/xstevenyung/epic-stack-with-fathom-analytics)
   by [@xstevenyung](https://github.com/xstevenyung): An example of the Epic Stack
   with Fanthom Analytics via CDN
+- [Epic Stack + Tenant Users](https://github.com/offseat/epic-stack-tenant)
```

Comment:
I think this may be private:

<img width="969" alt="image" src="https://github.com/epicweb-dev/epic-stack/assets/1500684/b85b1678-bcd8-40bb-bbf4-bc68e684977e">


Could you make it public?

Reply:
doh, public now!

Reply:
Thanks!

Unfortunately, it's hard to tell what changes are required for this example. The instructions say you should make a commit as soon as you initialize the project and then after you make your changes so people can see the difference between a standard epic stack project and one with the changes you make.

Could you do that please?

Reply:
Hi, yeah I'll update that

Reply:
Hi Kent,

I've updated it now

Reply:
Thanks!

I'm concerned the implementation is overly complicated. What do you think about doing this instead:

1. create a tenant model with an id and name
2. give all models a `tenantId` and relate them to the tenant (one-to-many relationship)
3. adjust queries to scope them to a specific `tenantId` (may require updating the routes to include the `tenantId` in the params)

I think that would be simpler than having a `User`, `TenantUser`, `WorkspaceUser` and `Member` all of which kinda feel like the same abstract concept to me. What do you think?

Reply:
Yeah they do seem to be sort of the same at the moment, though I chose to add extraction so when I want to query a WorkspaceUser I can simply do `prisma.workspaceuser`, a TenantUser is usually the admin of the tenant and manages subscription/payment, Members are the Individuals that belong to the Tenant.

It's a complex system that does need thorough documentation

Reply:
Ok, sounds good. Thanks!